docs/v1.md

```diff
@@ -207,60 +207,61 @@
 - type() : `enum{'ClientSubscribing', 'ClientPublishing', 'ApplicationSubscribing', 'ApplicationPublishing}`
 
 ## Schema
-- $id() : `string`
-- additionalItems : `Schema`
+- $id() : `string` | `undefined`
+- additionalItems : `boolean | Schema`
 - additionalProperties : `boolean | Schema`
-- allOf : `Schema`
-- anyOf : `Schema`
+- allOf : `Schema[]`
+- anyOf : `Schema[]`
 - circularProps : `string[]`
-- const : `any`
+- const : `string` | `undefined`
```

Comment:
I noticed this is not `any` but string as I saw in the JSON Schema spec.

Reply:
For `const` you can also define number value or boolean, so I would leave `any`

Reply:
Yeah good catch. Rolling back!